backlog/models.py

```diff
@@ -141,6 +141,17 @@ class Meta:
     rt_ticket = models.IntegerField(unique=True)
 
 
+class AssemblyType(models.Model):
+    class Meta:
+        db_table = 'AssemblyType'
+        app_label = 'backlog'
+
+    assembly_type = models.CharField(max_length=80, unique=True, null=False)
+
+    def __str__(self):
+        return self.assembly_type
+
+
 # Assemblies received from ENA
 class Assembly(models.Model):
```

Comment:
This model has several FKs (they are lazy loaded by default). My guess is that all that info will be loaded and used most of the time, to improve the performance I would add a custom Manager like this:

```python
class AssemblyManager(models.Manager):

    def get_queryset(self):
        qs = super().get_queryset()
        return qs.select_related('study', 'biome', 'inferred_biome', 'assembly_type')
```